src/scales/scale.time.js

```diff
@@ -488,6 +488,7 @@ module.exports = Scale.extend({
 		var me = this;
 		var options = me.options;
 		var time = options.time || (options.time = {});
+		var adapter = me._adapter = new adapters._date(time.adapter);
```

Comment:
I don't think the adapter options should be located under `options.time.adapter`.

@kurkle suggested under `options.adapters.date` which may be better?! 🤔

or maybe `options.date.adapter`?

Reply:
You had suggested earlier that you might want different adapter options for each time scale. Do you think the time options should be set at the chart-level instead? I can't think of a reason to have different time options for different scales, but it seemed more flexible for possibly allowing it if there was some use case we hadn't considered

Reply:
> You had suggested earlier that you might want different adapter options for each time scale

I'm still suggesting to do that: in my previous comment, `options` refers to `scale.options`

> Do you think the time options should be set at the chart-level instead?

No, but that's a good question. For now, I think we should keep this date logic at the scale level because we don't really know how this feature will evolve. So in case we move the time scale out of this repository, it would be great to not be locked with the date lib in the core build.

My comment was about what option name/structure to adopt under the scale options:

```javascript
new Chart(ctx, {
  options: {
    scales: {
      xAxes: [{
        adapters: { date: { ... }},  // Jukka suggestion
        time: { adapter: { ... }},   // your suggestion
        date: { adapter: { ... }},   // another idea
    }]
  }
});
```

Reply:
Ah, ok. That makes more sense. Thanks for clarifying.

I was suggesting `time.adapter` because we're aggregating all the time options under `time` already. I like `date.adapter` the least because we'd have `date` and `time` side-by-side. The one thing I like about `adapters.date` is that if we introduced other adapters at the scale-level we'd have a consistent place to put them. I'm not sure if we'll do that enough to warrant it especially since the library has no other dependencies.

Reply:
I think I prefer `adapters.date` since most `time` options should be moved, at some point, either under `ticks` or `adapters.date`.

@kurkle thought?

Reply:
I too prefer `adapters.date`.
If it would fall back some day (if used for something other in addition to time scale, for example):
`dataset.adapters.date`, `options.adapters.date`, `defalts.adapters.date` would be acceptable, but `time` would be kind of out of scope in those.

Reply:
Ok. Updated